pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>${zendesk-java-client.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-etl-api</artifactId>
+      <version>6.6.0-SNAPSHOT</version>
+    </dependency>
```

Comment:
Why are we using two different versions for the artifact `cdap-etl-api` ?

Reply:
Also, we shouldn't be using the SNAPSHOT version.

Reply:
Done the change.